lib/lev/object.rb

```diff
@@ -7,7 +7,24 @@ def self.lev_routine(options={})
       # Routine configuration
       options[:transaction] ||= Lev::TransactionIsolation.mysql_default.symbol
       @transaction_isolation = Lev::TransactionIsolation.new(options[:transaction])
-      @express_output = options[:express_output] || self.name.demodulize.underscore
+
+      @@delegates_to = options[:delegates_to]
+      if @@delegates_to
+        uses_routine @@delegates_to,
+                     translations: {
+                       outputs: { type: :verbatim },
+                       inputs: { type: :verbatim }
+                     }
+
+        @express_output ||= @@delegates_to.express_output
+
+        def exec(*args, &block)
+          run(@@delegates_to, *args, &block)
+        end
+      end
+
+      # Set this after dealing with "delegates_to" in case it set a value
+      @express_output ||= options[:express_output] || self.name.demodulize.underscore
     end
```

Comment:
Feature suggestion: Why not make delegates_to optionally accept an array?  That way you could do:

    lev_routine delegates_to: BarRoutine, FooRoutine

And it would call each routine in the order given.

This might be a bit much, but you might also support:

    lev_routine delegates_to: BarRoutine, self, FooRoutine

which would run Bar, the routine itself, and then Foo